modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2157,6 +2157,16 @@ protected CompletableFuture<Void> handleChangeStableAssignmentEvent(WatchEvent e
 
         return metaStorageMgr.get(pendingPartAssignmentsKey(tablePartitionId), stableAssignmentsWatchEvent.revision())
                 .thenComposeAsync(pendingAssignmentsEntry -> {
+                    CompletableFuture<Void> raftClientUpdateFuture = completedFuture(null);
+
+                    if (!stableAssignments.equals(ByteUtils.fromBytes(evt.entryEvent().oldEntry().value()))) {
+                        raftClientUpdateFuture = tablesById(evt.revision()).thenAccept(t -> {
```

Comment:
Please add some comments about the purpose of this code

Reply:
Done